cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -67,11 +68,17 @@ def _delete_asset(course_key, asset_key_string):
     remove asset by calling delete_asset method of assets module.
     """
     if asset_key_string:
-        # remove first slash in asset path
-        # otherwise it generates InvalidKeyError in case of split modulestore
-        if '/' == asset_key_string[0]:
-            asset_key_string = asset_key_string[1:]
-        asset_key = AssetKey.from_string(asset_key_string)
+        try:
+            asset_key = AssetKey.from_string(asset_key_string)
+        except InvalidKeyError:
```

Comment:
oh, snaps it looks like diff-cover did catch it but didn't fail the edx-platform-unit-coverage job. 
Might be a bug in [jenkins-report.sh](https://github.com/edx/edx-platform/blob/master/scripts/jenkins-report.sh) as we are not checking the result of the paver coverage command. 
From https://build.testeng.edx.org/job/edx-platform-unit-coverage/27064/console
```
21:59:19 -------------
21:59:19 Diff Coverage
21:59:19 Diff: origin/master...HEAD, staged, and unstaged changes
21:59:19 -------------
21:59:19 cms/djangoapps/contentstore/views/certificates.py (37.5%): Missing lines 73,76-78,80
21:59:19 cms/djangoapps/contentstore/views/preview.py (100%)
21:59:19 cms/djangoapps/contentstore/views/tests/test_certificates.py (100%)
21:59:19 cms/lib/xblock/tagging/test.py (100%)
21:59:19 common/lib/xmodule/xmodule/contentstore/content.py (100%)
21:59:19 common/lib/xmodule/xmodule/x_module.py (100%)
21:59:19 lms/djangoapps/courseware/model_data.py (100%)
21:59:19 -------------
21:59:19 Total:   50 lines
21:59:19 Missing: 5 lines
21:59:19 Coverage: 90%
21:59:19 -------------
```